README.md

```diff
@@ -0,0 +1,60 @@
+# inumpy: Introduction to NumPy Using iPython Notebook
+
+## Synopsis
+
+This one-day workshop introduces [NumPy](http://www.numpy.org/), the
+fundamental library for numerical computing in Python, and how to use its
+building blocks.  We will use [iPython](http://ipython.org/) (or equivalently,
+[Jupyter](https://jupyter.org/)) notebook, an excellent interactive computing
+system, as the key interface to the numerical library.  Although iPython is a
+great companion of NumPy, the former is not only useful in the numerical
+computing.
+
+## Prerequisites
+
+Attendees should be familiar with Python and Unix-based terminal operations
+(command-line user interface).  Attendees should bring their own computers.
+The workshop will prepare a [VirtualBox](https://www.virtualbox.org/) virtual
+machine image of Ubuntu Linux that is loaded with the necessary software
+packages and the workshop materials.
+
+## Instructors:
+
+- Albert Huang
+- Tzer-jen Wei
+- [Yung-Yu Chen](mailto:yyc@solvcon.net)
+
+## (Tentative) Contents
+
+### iPython
+
+- Environment preparation
+- Terminal mode
+- Notebook
+  - Slides
+- Administration
+
+### NumPy Basic Operations
+
+- The look and feel of ndarray
+  - Visualization: matplotlib
+- Creation of ndarray
+- Manipulation of ndarray
+- Iteration of ndarray
+- Broadcasting
+- Slicing
+
+### NumPy Computing
+
+- Ufunc
+  - Performance
+- Linear algebra
+- Discrete Fourier transform
+
+### NumPy C Extension
+
+- Cython
+
+### Problems (Hands-on)
```

Comment:
I think hands-on problem solving should be blended into the above topics.  But at this early stage I don't know how to do it yet.